sale_order_product_recommendation_elaboration/wizards/sale_order_recommendation.py

```diff
@@ -38,21 +38,26 @@ class SaleOrderRecommendationLine(models.TransientModel):
     _name = "sale.order.recommendation.line"
     _inherit = ["sale.order.recommendation.line", "product.elaboration.mixin"]
 
-    def _prepare_elaborations_line_form(self, line_form):
+    def _prepare_elaborations_line_form(self, vals):
         """Transfer elaborations to a sale order line form."""
-        line_form.elaboration_ids.clear()
-        for elaboration in self.elaboration_ids:
-            line_form.elaboration_ids.add(elaboration)
-        line_form.elaboration_note = self.elaboration_note
+        if self.sale_line_id:
+            self.sale_line_id.update({"elaboration_ids": [(fields.Command.clear())]})
```

Comment:
Why are you changing here directly the sale line?

Reply:
Hi @pedrobaeza, do you think it's better to remove it ?
As I understood, elaboration_ids will be updated anyway at the end.